src/main/fish/payara/server/PayaraInstanceController.ts

```diff
@@ -26,13 +26,14 @@ import * as fs from "fs";
 import * as tmp from "tmp";
 import * as fse from "fs-extra";
 import * as cp from 'child_process';
+import * as xml2js from 'xml2js';
```

Comment:
Duplicate import, see line 24.

Reply:
It looks like a few merge commits went a bit funky, this needs sorting before a review